### DISCUSSION TITLE
Version Control and Timeline

### DISCUSSION THREAD

**Discussion by 000347334** [Leigh-S/Tafe24S2T2EXT01#2] [General]
Finally getting time to work on the Currency converter. 

I will fork from the development branch while I build and test the code. 

Can I push it back to the development branch or do I need to submit a pull request?

(Git's push / pull jargon confuses me)

**Comment by 000347334** [comment]
I am aware this project needs to be finished soon, do you have any any landmarks or deadlines we need to work to?

**Reply by Leigh-S** [reply]
Na, I think we're looking good to have it completed by the weekend.

**Comment by Leigh-S** [comment]
> Finally getting time to work on the Currency converter.
> 
> I will fork from the development branch while I build and test the code.
> 
> Can I push it back to the development branch or do I need to submit a pull request?
> 
> (Git's push / pull jargon confuses me)

you should be able to push direct from TortioseGit (I think)
